lldb/source/Target/Target.cpp

```diff
@@ -1537,6 +1538,154 @@ static void LoadScriptingResourceForModule(const ModuleSP &module_sp,
                                                   feedback_stream.GetData());
 }
 
+static void ForEachFormatterInModule(
```

Comment:
Target seems like a weird place for this code to live. I understand that's where `LoadScriptingResourceForModule` currently resides, but that function is pretty basic. I think we've reached the threshold of having to move `LoadScriptingResourceForModule`, `LoadTypeSummariesForModule` and `LoadFormattersForModule` into a separate file.